instaclustr/resource_cluster.go

```diff
@@ -584,6 +587,13 @@ func resourceCluster() *schema.Resource {
 		},
 	}
 }
+
+func versionDiffSuppressFunc(k, old string, new string, d *schema.ResourceData) bool {
```

Comment:
might be nice to have a comment explaining exactly what this is doing?  Im not a regex expert so i can only guess.

Reply:
Yep, can do.